src/include/ceph_fs.h

```diff
@@ -350,9 +350,10 @@ enum {
 	// internal op
 	CEPH_MDS_OP_FRAGMENTDIR= 0x01500,
 	CEPH_MDS_OP_EXPORTDIR  = 0x01501,
-	CEPH_MDS_OP_VALIDATE   = 0x01502,
-	CEPH_MDS_OP_FLUSH      = 0x01503,
-	CEPH_MDS_OP_ENQUEUE_SCRUB = 0x01504
+	CEPH_MDS_OP_FLUSH      = 0x01502,
+	CEPH_MDS_OP_ENQUEUE_SCRUB  = 0x01503,
+	CEPH_MDS_OP_REPAIR_FRAGSTATS = 0x01504,
+	CEPH_MDS_OP_REPAIR_INODESTATS = 0x01505
```

Comment:
Mmm, granted none of these have been exported yet that I can think of, but it's not a good idea to change these numbers since as a group they are shared outside the local daemon. Probably should just change CEPH_MDS_OP_VALIDATE to CEPH_MDS_OP_DEPRECATED and leave the actual numerical values the same.